pyvista/py.typed

```diff
@@ -0,0 +1 @@
+partial\n
```

Comment:
```diff
-partial\n
+partial
```

https://peps.python.org/pep-0561/#partial-stub-packages

https://github.com/python/mypy/blob/cc8a4b50f7e65004a97c9ba51c69f7c9340370d9/test-data/packages/typedpkg-stubs/typedpkg-stubs/py.typed#L1

https://github.com/python-poetry/poetry/blob/02448cf7f184dea204156f7dcb620a4f01a0068e/tests/masonry/builders/fixtures/pep_561_stub_only_partial/pkg-stubs/py.typed#L1

Reply:
> If a stub package distribution is partial it MUST include partial\n in a py.typed file.

I believe it needs a  newline. We can remove the `\n`, but the PEP seems to identify the file must end with a new line

Reply:
I’ve posted the question on the Python Discussion Forum:

https://discuss.python.org/t/pep-561-clarification-regarding-n/32875

My main question is whether the `\n` is deliberately required in order to distinguish Linux/MacOS line endings from Windows ones (`\r\n`) (albeit, which can be addressed with `git config`) and whether the `\n` is literally required.

Reply:
@eggplants Per [the discussion](https://discuss.python.org/t/pep-561-clarification-regarding-n/32875), this works with or without the newline in the [`mypy` implementation](https://github.com/python/mypy/blob/c7d2fa1525c9cbf0ab8859fd9ded526658677c28/mypy/modulefinder.py#L439) since that checker strips all whitespace, including explicit escape sequences (I have not looked at `pyre`, `pyright`, or other type checkers.

It seems a new line is requested to be included by convention, specifically the POSIX convention.

This works with the `\n` (the package is registered as partially typed), so nothing is breaking.

It’s a “6 one way, half a dozen the other” problem, so if we want this changed, I recommend removing the escape sequence, but adding a new line to the end of the file (i.e. hit Enter/Return on your keyboard one more time).

Reply:
@adam-grant-hendry I've been trying to make it clear that this _shouldn't_ work. What we have in this PR doesn't include an escape sequence. The exact contents of the file are `'partial\\n\n'`. There's a literal backslash, a literal n, and then an actual linefeed character. If you strip away this content you get `'partial\\n'`. Odds are the typing "works" because of the "edge case" code path in the above mypy code (I haven't looked closely).

Reply:
Also note that
> I recommend removing the escape sequence, but adding a new line to the end of the file (i.e. hit Enter/Return on your keyboard one more time).

will likely give you _two_ line-feeds at the end. I.e. an additional empty line.

Reply:
> I've been trying to make it clear that this shouldn't work.

Then why are we “fixing” a non-problem?

Reply:
It is not worked with mypy.

https://github.com/python/mypy/blob/ed9b8990025a81a12e32bec59f2f3bfab3d7c71b/mypy/modulefinder.py#L439C4-L439C4

`fscache.read(stub_typed_file).decode().strip()` returns `'partial\\n'` and it is not recognized as Partial typed package because `'partial\\n' == 'partial'` returns `False`.

Reply:
> > I've been trying to make it clear that this shouldn't work.
> 
> Then why are we “fixing” a non-problem?

I can't really explain it in any other way. Please reread my comments (also on discuss) carefully. What "shouldn't work" is the file contents we have now. If it works now it's accidental. We are "fixing it" because it's wrong. The literal backslash and literal n must go. @eggplants' suggestion here is the fix to make it correct.

Reply:
> `fscache.read(stub_typed_file).decode().strip()` returns 'partial\\n' and it is not recognized as Partial typed package because 'partial\\n' == 'partial' returns False.

Okay, THAT's the answer I was looking for. Thank you for the clear explanation. If that's the case, then certainly, `partial\n` does not work.

Reply:
@eggplants Did you happen to open a separate PR for this?

Reply:
Just created! #4863